.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
should be removed

Reply:
Nice catch. Removed in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/183/commits/88c9319ad5cb3b91bab20e3891cba1233e78b246